types/airtable/index.d.ts

```diff
@@ -0,0 +1,56 @@
+// Type definitions for airtable 0.5
+// Project: https://github.com/airtable/airtable.js
+// Definitions by: Brandon Valosek <https://github.com/bvalosek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+export default class Airtable {
+    base(appId: string): Base;
+}
+
+export interface Base {
+    (tableName: string): Table<{}>;
+}
+
+export interface FieldSet {
+    [ key: string ]: undefined | string | ReadonlyArray<Attachment>;
+}
+
+export interface Table<TFields extends FieldSet> {
+    select(opt?: SelectOptions): Query<TFields>;
+}
+
+export interface SelectOptions {
+    view?: string;
+}
+
+export interface Query<TFields extends object> {
+    all(): Promise<Response<TFields>>;
+    firstPage(): Promise<Response<TFields>>;
+}
+
+export type Response<TFields> = ReadonlyArray<Row<TFields>>;
+
+export interface Row<TFields> {
+    id: string;
+    fields: TFields;
+}
+
```

Comment:
Because this package can be used in browser as well you want to define it as global as well. 
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.